apps/backend/src/mcp/server.py

```
@@ -1,470 +1,649 @@
 
-from fastmcp import FastMCP, Context
+# Add the project root to Python path
+project_root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))))
+sys.path.insert(0, project_root)
 
-# Import tool modules
-from apps.backend.src.mcp.tools.container_management import CONTAINER_TOOLS
-from apps.backend.src.mcp.tools.device_management import DEVICE_TOOLS
-from apps.backend.src.mcp.tools.system_monitoring import SYSTEM_MONITORING_TOOLS
+# Import device management functions
+from apps.backend.src.mcp.tools.device_management import add_device as device_add_device
```

Ruff (0.12.2)

Module level import not at top of file

(E402)